agri-store-react/src/Components/Header/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./Header.css";
+
+function Header() {
+  return (
+    <div>
+      <nav class="navigation-menu">
+        <div class="navigation__left">
+          <div class="navigation__logo">Agri UI</div>
```

Comment:
If possible use p tag or h1 tag here, it would be more semantic

Reply:
thanks for the suggestion will update the same